package.json

```diff
@@ -84,7 +86,7 @@
         "@semantic-release/git",
         {
           "assets": [
-            "./CHANGELOG.md",
+            "./**/*.md",
```

Comment:
This really freaks me out until we had a chance to test this with an actual release. For the scope of this PR can we just specify the actual `.md` docs that we expect to be committed back?